src/drivers/generic/dummy-dma.c

```diff
@@ -240,8 +240,8 @@ static struct dma_chan_data *dummy_dma_channel_get(struct dma *dma,
 		}
 	}
 	spin_unlock_irq(&dma->lock, flags);
-	trace_dummydma_error("dummy-dmac: %d no free channel",
-			     dma->plat_data.id);
+	tr_err(&ddma_tr, "dummy-dmac: %d no free channel",
+	       dma->plat_data.id);
```

Comment:
I think we no longer need the prefix "dummy-dmac: " on this one. Similarly in other places.